programmers/난이도별/level01.완주하지_못한_선수/sokkanji.js

```diff
@@ -0,0 +1,7 @@
+function solution(participant, completion) {
+    let answer='';
+    if(participant.length < 1 && participant.length > 100000) return;
+    participant.sort();
+    completion.sort();
+    return answer = participant.find((p, i) => p != completion[i]);
```

Comment:
```suggestion
    return participant.find((p, i) => p != completion[i]);
```
answer에 할당하는건 생략해도 될 것 같아요!

Reply:
앗 그렇네요!! 알려주셔서 감사합니다!!😊😊